src/screen-size-context-provider.tsx

```diff
@@ -93,10 +93,10 @@ const ScreenSizeContextProvider: React.FC<Props> = ({children}) => {
             isTablet,
             isTabletOrBigger,
```

Comment:
not related to your changes, but there's a typo in `mediaQueries.largueDesktop` (largue)

Reply:
I'll fix it

Reply:
done